src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/common/model/in/ExceptionRecord.java

```diff
@@ -29,7 +33,11 @@ public ExceptionRecord(
         @JsonProperty("delivery_date") LocalDateTime deliveryDate,
         @JsonProperty("opening_date") LocalDateTime openingDate,
         @JsonProperty("scanned_documents") List<InputScannedDoc> scannedDocuments,
-        @JsonProperty("ocr_data_fields") List<OcrDataField> ocrDataFields
+        @JsonProperty("ocr_data_fields") List<OcrDataField> ocrDataFields,
+        @JsonProperty("envelope_id") String envelopeId,
+        @JsonProperty("is_automated_process") String isAutomatedProcess,
+        @JsonProperty("exception_record_id") String exceptionRecordId,
```

Comment:
is that not the `id` property on top?

Reply:
yeah, it is `id` but it should be renamed as `exception_record_id` for the new endpoint. 
same for `case_type_id`. `case_type_id` will be renamed a s`exception_record_case_type_id`.
We have to keep all the above fields to support both old and new transformation endpoints

Reply:
ah rename process. ok, got you